lib/manager/components/deployment/DeploymentViewer.js

```diff
@@ -413,6 +414,12 @@ export default class DeploymentViewer extends Component<Props, State> {
               deployment={deployment}
               updateDeployment={updateDeployment}
             />
+            {/* Pelias panel. Only show if deployment is pinned */}
+            {project.pinnedDeploymentId === deployment.id && <PeliasPanel
```

Comment:
This always seemed like a weird design choice to me, but I guess it makes sense if there is just one instance that does this stuff for now.

Reply:
I think ideally data tools will deploy the Pelias server as well which will make this restriction redundant